### PR TITLE
refactor: Update to assign and uses new Port Assignments

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-mqtt-go/blob/master/.github/CONTRIBUTING.md

## What is the current behavior?
Ports assigned for Core/Supporting services are 408xx range
Device Modbus uses 49982

## What is the new behavior?
Ports assigned for Core/Supporting services are 598xx range
Device MQTT default port number has changed to 59982

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

BREAKING CHANGE: Device MQTT default port number has changed to 59982

## New Imports

- [ ] Yes
- [x] No

### CODE REVIEW

File: cmd/res/configuration.toml [hunk 1]
```diff
@@ -37,12 +37,12 @@ Type = 'consul'
   [Clients.core-data]
   Protocol = 'http'
   Host = 'localhost'
-  Port = 48080
+  Port = 59780
```
Comment (hahattan) [inline cmd/res/configuration.toml:40]: 59880

File: cmd/res/configuration.toml [hunk 2]
```diff
@@ -37,12 +37,12 @@ Type = 'consul'
...
 
   [Clients.core-metadata]
   Protocol = 'http'
   Host = 'localhost'
-  Port = 48081
+  Port = 59781
```
Comment (hahattan) [inline cmd/res/configuration.toml:45]: 59881

File: mock/device.go [hunk 3]
```diff
@@ -32,12 +32,12 @@ func main() {
...
-// http://localhost:49982/api/v1/devices/{device id}>/message - use POST on this one with
-// {"message":"some text"} in body http://localhost:49982/api/v1/devices/<device id>/ping - use GET
-// http://localhost:49982/api/v1/devices/<device id>/randnum - use GET
+// http://localhost:59982/api/v1/devices/{device id}>/message - use POST on this one with
+// {"message":"some text"} in body http://localhost:59982/api/v1/devices/<device id>/ping - use GET
+// http://localhost:59982/api/v1/devices/<device id>/randnum - use GET
 //
 // If command micro service is running, the same can be performed through command to device service
-// like this http://localhost:48082/api/v1/device/<device id>/command/<command id>
+// like this http://localhost:59782/api/v1/device/<device id>/command/<command id>
```
Comment (hahattan) [inline mock/device.go:40]: 59882

**Review by hahattan — CHANGES_REQUESTED** [review]

File: cmd/res/configuration.toml (hunk 1 above)
Comment (lenny-intel) [reply cmd/res/configuration.toml:40]: Good catch!

Comment (lenny-intel) [reply cmd/res/configuration.toml]: fixed

File: cmd/res/configuration.toml (hunk 2 above)
Comment (lenny-intel) [reply cmd/res/configuration.toml]: fixed

File: mock/device.go (hunk 3 above)
Comment (lenny-intel) [reply mock/device.go]: fixed

**Review by hahattan — APPROVED** [review]
LGTM

**Pull request merged**